README.md

```diff
@@ -37,14 +37,65 @@ The `ContractTest` class demonstrates how to use Specmatic to test **specmatic-o
    git config submodule.recurse true
    ```
 
-4. To run contract tests, execute
+## Running Contract Tests
+
+### Using Gradle
 
    ```shell
    ./gradlew clean test   
    ```
 
-5. In case you want to run just the gRPC server using Gradle you can execute
+### Using Docker
+
+1. Start the Specmatic gRPC stub server to emulate domain service:
+
+   ```shell
+   docker run -p 9090:9090 -v "$(pwd)/specmatic.yaml:/usr/src/app/specmatic.yaml" znsio/specmatic-grpc-trial stub --port=9090
+   ```
+
+2. Build and run the BFF service (System Under Test) in a Docker container:
+
+   ```shell
+   docker build --no-cache -t specmatic-order-bff-grpc .
+   ```
+
+   ```shell
+   docker run --network host -p 8080:8080 specmatic-order-bff-grpc
+   ```
+
+3. Finally, run Specmatic Contract on the BFF service (System Under Test):
+
+   ```shell
+   docker run --network host -v "$(pwd)/specmatic.yaml:/usr/src/app/specmatic.yaml" znsio/specmatic-grpc-trial test --port=8080
+   ```
```

Comment:
The test failed for me on first run but passed from second run onwards.
Shall we add a note here for that? Something around `If one of the test fails due to DEADLINE_EXCEEDED error, try running it again` ? cc - @pranavgawri